Mlem/Icons.swift

```diff
@@ -149,6 +149,12 @@ enum Icons {
     static let close: String = "multiply"
     static let cakeDay: String = "birthday.cake"
     
+    // uptime
+    static let uptimeOffline: String = "xmark.circle.fill"
+    static let uptimeOnline: String = "checkmark.circle.fill"
+    static let uptimeShortOutage: String = "ellipsis.circle.fill"
```

Comment:
This symbol choice feels odd. I initially tapped it thinking it would be a button that would show me more info or something like that. Exclamation mark wouldn't be great, though--we could maybe go the Jenkins route and use cloud/rain/thunder for severity?

Reply:
Sure, I think the exclamation mark would look good 👍